src/components/organizations/newOrganizationFields.ts

```diff
@@ -0,0 +1,10 @@
+import { nestedFieldNamed } from "../forms/nestedFieldNamed";
+import { mapProperties } from "../../utils/list";
+import { generalOrgFields } from "./generalOrgFields";
+
+export const newOrganizationFields = 
```

Comment:
here we use the same `generalOrgFields` but instead of the more complex nesting within the matching `state.organizations` with matching href; we need only have the fields nested within something simpler - the `state.possibleNewOrganization` object. this is how we define such a structure